content/_index.html

```diff
@@ -15,8 +15,11 @@
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="/FSHOnline/">
 		Play with FSH <i class="fas fa-fish "></i>
```

Comment:
```suggestion
		Play With FSH <i class="fas fa-fish "></i>
```

Title case fix (see comment below).